docs/ota-client-guide/modules/ROOT/pages/build-raspberry.adoc

```diff
@@ -70,7 +71,7 @@ Now you can run the following script to get the environment set up:
 
 [subs=+attributes]
 ----
-source meta-updater/scripts/envsetup.sh {machine}
+source meta-updater/scripts/envsetup.sh {meta-env}
```

Comment:
This one should still use `machine`. The other change above (referring to `meta-updater-raspberrypi`) is correct, though.

Reply:
fixed